ui/src/views/Rules/ListRules/index.jsx

```diff
@@ -1235,6 +1316,7 @@ function ListRules(props) {
             : Object.values(rule.scheduledChange).join('-')
         }
         style={style}>
+        {/* should we go read only mode if rewindDate is set instead? */}
```

Comment:
todo: please remove this comment. (It was an interesting thought, but I did a quick experiment locally and I don't think using readOnly mode is as nice as simply disabling the controls as you're doing already.)